project-directory.md

```diff
@@ -116,90 +116,94 @@ These are all the repos found in the IPFS Github organization. If you still can'
     Examples on how to use go-ipfs.
 
 ## Community Resources
+
+* [awesome-ipfs](https://github.com/ipfs/awesome-ipfs)  
+    Useful resources for using IPFS and building things on top of it.
+
+* [blog](https://github.com/ipfs/blog)  
+    IPFS Blog.
+
 * [community](https://github.com/ipfs/community)  
     General discussion and documentation on community practices.
 
-* [support](https://github.com/ipfs/support)  
-    For questions on how to get IPFS up and running smoothly.
-
 * [infrastructure](https://github.com/ipfs/infrastructure)  
     Tools and systems for the community.
 
+* [logo](https://github.com/ipfs/logo)  
+    The logo for IPFS.
+
+* [meetup-lisbon](https://github.com/ipfs/meetup-lisbon)  
```

Comment:
I guess this is already available on the community repo. If we decide to replicate it here, we should replicate all of the meetups.